### ISSUE TITLE
On startup, server should complete existing jobs before adding new ones

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#608]
**What would you like to be added**:

When we start the server, one of the first things we do is to download the wiki feeds and create feed jobs in our job queue.  We do this regardless of whether or not there are already jobs in the queue for this feed's URL.

Let's allow the queue to `drain` existing jobs before we add our first batch of feed jobs.  This will let the server pick-up where it left off.

**Why would you like this to be added**:

The way things work now, it's easy to get thousands, or tens-of-thousands of jobs in the queue without realizing it.

To see what I mean, follow these steps:

* Start the server `npm start`
* Open your browser to `http://localhost:3000/admin/queues` so you can see a live view of what's in the feed queue
* Notice the number of jobs in the `waiting` area of the queue.  It should be something like 500-700 or less, for sure less than 1,000.  Here's what mine looks like when I do this:

<img width="1126" alt="Screen Shot 2020-01-26 at 11 32 05 AM" src="https://user-images.githubusercontent.com/427398/73138402-3335e780-4030-11ea-9580-b59a7e4d2965.png">

* Now kill the server with `ctrl+c` and then restart it.  Look at the jobs again:

<img width="1118" alt="Screen Shot 2020-01-26 at 11 32 17 AM" src="https://user-images.githubusercontent.com/427398/73138401-3335e780-4030-11ea-805a-3fc66353eb82.png">

We can use [`const jobCounts = await queue.getJobCounts()`](https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md#queuegetjobcounts) or [`const waitingJobsCount = await queue.getWaitingCount()`](https://github.com/OptimalBits/bull/blob/develop/REFERENCE.md#queuegetwaitingcount) to figure out how many jobs are waiting in the queue right now, and if it's > 0, we should wait wait on the queue to be fully `drained` (i.e., empty) before we add our new jobs.

**Comment by humphd** [comment]
Looking at this on http://dev.telescope.cdot.systems/admin/queues, it seems like we may need to take into account all jobs vs. just waiting job count, since we don't want to flood the machine with thousands (or 10s of thousands) of delayed jobs, like we do now.

Monitoring the flow of new jobs into the queue will help things a lot here.

**Comment by humphd** [comment]
@Silvyre I think I accidentally fixed part of this while working on another issue, see #618.  Sorry about that.
